source/default_mode/Symbiont.h

```diff
@@ -543,6 +551,10 @@ class Symbiont: public Organism {
       emp::Ptr<Organism> sym_baby = Reproduce();
       points = points - my_config->SYM_VERT_TRANS_RES();
       host_baby->AddSymbiont(sym_baby);
+
+      //vertical transmission data node
+      emp::DataMonitor<int>& data_node_attempts_verttrans = my_world->GetVerticalTransmissionAttemptCount();
```

Comment:
Do we need to get it each time or can we make sure its initialized somewhere else and just use the instance variable?